src/rurl/url.c

```diff
@@ -158,7 +162,7 @@ int rc_url_get_badge_image(char* buffer, size_t size, const char* badge_name) {
   int written = snprintf(
     buffer,
     size,
-    "http://i.retroachievements.org/Badge/%s",
+    RCHEEVOS_URL_PROTOCOL"i.retroachievements.org/Badge/%s",
```

Comment:
The image host doesn't currently support HTTPS. This one shouldn't change.

Reply:
``` suggestion
    "http://i.retroachievements.org/Badge/%s",
```

Reply:
Thanks!